docs/concepts/namespaces.md

```diff
@@ -33,3 +33,16 @@ Examples:
 
 - `gno.land/p/demo/avl`: This signifies a package named `avl` within the `demo` namespace.
 - `gno.land/r/gnoland/home`: This signifies a realm named `home` within the `gnoland` namespace.
+
+## Registration Process
+
+The registration process is contract-based. The `AddPkg` command references `sys/users` for filtering, which in turn is based on `r/demo/users`.
+
+When `sys/users` is enabled, you need to register a name using `r/demo/users`. This process requires an `Invite`, after which you can call the `r/demo/users.Register` function to register the name for the caller's address.
```

Comment:
Does it require an invite every time?

I think you can just put down a deposit without an invite and it will work, but with an invite it's free -- we should note this here

Reply:
We should also give an example here

Reply:
yea i forgot to update the description